package.json

```diff
@@ -3,14 +3,19 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "start": "node server.js",
-    "watch": "npm run watch:server & npm run watch:client",
-    "watch:server": "nodemon server.js",
-    "watch:client": "webpack -w --progress --colors",
-    "dev": "webpack-dev-server",
-    "build": "NODE_ENV=production webpack -p",
+    "start": "node build/server.js",
+    "build": "mkdir -p build && npm run build:static && npm run build:client && npm run build:server",
+    "build:client": "webpack --progress --config webpack.client.config.js",
+    "build:production": "export NODE_ENV=production && npm run build",
+    "build:server": "webpack --config webpack.server.config.js",
+    "build:static": "cp -R data ./build/data && cp -R img ./build/img && cp robots.txt ./build",
+    "watch": "mkdir -p build  npm run build:static && npm run watch:server & npm run watch:client",
+    "watch:server": "npm run watch:server:build & npm run watch:server:run",
```

Comment:
can we simplify `watch:server` to just be `webpack -w --config webpack.server.config.js & nodemon build/server.js` and drop the `watch:server:build` and `watch:server:run` scripts -- there's just too many!